code/modules/clothing/gloves/_gloves.dm

```diff
@@ -40,13 +40,20 @@
 	if(ismob(loc))
 		var/mob/M = loc
 		M.update_worn_gloves()
-
-/obj/item/clothing/gloves/wirecutter_act(mob/living/user, obj/item/I)
-	. = ..()
+		
+/obj/item/clothing/gloves/proc/can_cut_with(obj/item/I, mob/user)
 	if(!cut_type)
-		return
+		return FALSE
 	if(icon_state != initial(icon_state))
-		return // We don't want to cut dyed gloves.
-	new cut_type(drop_location())
-	qdel(src)
-	return TRUE
+		return FALSE // We don't want to cut dyed gloves.
+	else
+		return TRUE
```

Comment:
```suggestion

	return TRUE
```

Same thing but makes it easier to add new conditions and more cleanly separates them